specifications/api/swagger.yaml

```diff
@@ -1212,6 +1230,27 @@ x-transactionResolvedInputs: &transactionResolvedInputs
         type: integer
         minimum: 0
 
+x-transactionResolvedCollateral: &transactionResolvedCollateral
+  description: A list of transaction inputs used for collateral
```

Comment:
What does "resolved" here mean?

Reply:
> What does "resolved" here mean?

Good question. I'm following the existing terminology for inputs, as much as possible.

I think this means that in addition to including the `txId` and `txIndex` (which is enough to uniquely identify an input), we also include the following information, which shows what this input "resolves", or "maps" to:
- the `address`
- the `derivationPath`
- the `amount`